components/MainNavigation.vue

```diff
@@ -19,7 +19,12 @@
         :label="$t('profiles.title')"
         type="nav"
       />
-      <TButton to="/chat" icon="chat" :label="$t('chat.title')" type="nav" />
+      <TButton
+        to="/chat"
+        icon="chat"
+        :label="$t('myprofile.chat.label')"
```

Comment:
That was correct key `chat.title` for Main Navigation, same key is used in the `chat/index.vue`. That has nothing to do with "My Profile".